korgan/professional_consultation_guard.py

```
@@ -0,0 +1,253 @@
+    "required": ["summary", "analysis", "recommended_actions", "verified_points", "unverified_claims"],
+    "additionalProperties": False,
+}
+
+# Precise law belongs only to the source-bound verified block below. These
+# patterns deliberately catch article numbers, legal rates/MRP and exact legal
+# periods that a free-form model sentence could otherwise smuggle into the
+# answer without a verified source binding.
+_PRECISE_LAW_RE = re.compile(
+    r"(?i)(?:"
+    r"(?:стать[ьяеию]\w*|ст\.)\s*\d+"
+    r"|\b\d+(?:-\d+)?\s*[-–]?\s*бап\b"
```

Ruff (0.16.1)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)